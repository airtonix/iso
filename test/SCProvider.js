const React = require('react')
const { create: render } = require('react-test-renderer')
const SCProvider = require('../src/SCProvider').default

describe('SCProvider', () => {
  test('renders', () => {
    const json = render(
      React.createElement(SCProvider, {},
        React.createElement('h1', null, 'hello')
      )
    ).toJSON()
    expect(json).toMatchSnapshot()
  })
})
