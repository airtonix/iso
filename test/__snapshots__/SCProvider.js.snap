// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SCProvider renders 1`] = `
<div
  className="sc-bdVaJa cUmmwg"
  color="#000"
>
  <h1>
    hello
  </h1>
</div>
`;
